extensions/iam/distributed-identity/registration-service/doc/onboarding_flow.md

```diff
@@ -0,0 +1,32 @@
+## Preconditions, assumptions, terminology
+
+- It is assumed that a DID is already in place and publicly available. The generation or deployment of a DID is not part
+  of this flow.
+- The terms "connector" and "participant agent" are to be used interchangeably
```

Comment:
You may want to distinguish between a "single instance connector" and "connector" in the sense here. A "connector" is basically one or more replicas, which share a common identity.

Reply:
good point, will amend.